.packit.yaml

```diff
@@ -33,6 +33,32 @@ packages:
         - bash -c "ls -1t ./fedora/*.tar.gz | head -n 1"
       get-current-version: python3 -m setuptools_scm
 
+  specfile-centos-stream:
+    upstream_package_name: specfile
+    downstream_package_name: python-specfile
+    specfile_path: &specfile_path_centos centos-stream/python-specfile.spec
+    files_to_sync:
+      - *specfile_path_centos
+      - .packit.yaml
+      - src: plans/
+        dest: plans/
+      - src: .fmf/
+        dest: .fmf/
+    srpm_build_deps:
+      - python3-build
+      - python3-setuptools_scm
+    actions:
+      pre-sync:
+        - python3 plans/git_reference.py
+      create-archive:
+        - python3 -m build --sdist --outdir ./fedora/
+        - bash -c "ls -1t ./centos-stream/*.tar.gz | head -n 1"
+      get-current-version: python3 -m setuptools_scm
+    dist_git_base_url: https://gitlab.com # this is not taken into account by centpkg-sig yet!
```

Comment:
> this is not taken into account by centpkg-sig yet!

So this needs some changes in Packit as well for these options to be passed to `centpkg-sig` (or the configured `pkg_tool` in general)?

Reply:
Yes exactly, this is not working yet. Without changes in packit and centpkg-sig.